.circleci/ca-circleci-key.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
acn we somehow mark the flle as no secret? Actually is it a secret?

Reply:
It's not a secret because is a fake cert I created for circleci, I marked it as non-secret using `rh-gitleaks` (https://github.com/3scale/apisonator/pull/385/commits/b1a50d44859008697bfd84e294768441c81691fa#diff-f40ac94aac00bf364ae4ebef5689f7a2b61d63788030902c529d48d6d7d30ffc).

Check https://source.redhat.com/departments/it/it-information-security/wiki/pattern_distribution_server#handling-false-positives